core/src/test/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionSpec.scala

```diff
@@ -518,14 +526,14 @@ class WorkflowSubmissionSpec(_system: ActorSystem)
   }
 
   it should "resolve DOS URIs when submitting workflows" in withDefaultTestDatabase {
```

Comment:
I think we can just get rid of the three resolve URI tests in favor of the new one where I only check the behavior of resolveDrsSignedUrls.  Is there anything else being exercised here that needs these tests?